src/intents.cs

```diff
@@ -2294,6 +2806,13 @@ interface INInteraction : NSSecureCoding, NSCopying {
 
 		[NullAllowed, Export ("groupIdentifier")]
 		string GroupIdentifier { get; set; }
+
+		// From INParameter.h INInteraction ()
+
+		[iOS (11,0), Watch (4,0), NoMac]
+		[Export ("parameterValueForParameter:")]
+		[return: NullAllowed]
+		NSObject GetParameterValue (INParameter parameter);
```

Comment:
That's _generally_ a breaking change and not a good candidate for overload (in C#) since only the return type is different